markdown/deployment/src/main/java/io/quarkiverse/qute/web/markdown/deployment/MarkdownConfig.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkiverse.qute.web.markdown.deployment;
+
+import java.util.Map;
+
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigRoot
+@ConfigMapping(prefix = "quarkus.qute.web.markdown")
+public interface MarkdownConfig {
+
+    /**
+     * The plugin configuration. Allowed keys are <code>autolink</code> and <code>heading-anchor</code>.
+     */
+    Map<String, PluginConfig> plugins();
```

Comment:
Since some might be enabled by default and some other might not (and there a not thousand of them), I would use a specific config per plugin (instead of a map):
```properties
quarkus.qute.web.markdown.plugin.tables.enabled=true
```